server/tests/services/correlation/test_correlation_outcome.py

```
@@ -0,0 +1,96 @@
+_server_dir = os.path.join(os.path.dirname(__file__), os.pardir, os.pardir, os.pardir)
+if os.path.abspath(_server_dir) not in sys.path:
+    sys.path.insert(0, os.path.abspath(_server_dir))
+
+from services.correlation.alert_correlator import (  # noqa: E402
+    CorrelationResult,
+    apply_correlation_outcome,
+    attach_correlation_hint,
+)
+
+
+def _call(mode, alert_metadata):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_call`

(ANN202)